selfdrive/car/body/carstate.py

```diff
@@ -57,4 +59,19 @@ def get_can_parser(CP):
       ("BODY_DATA", 1),
     ]
 
+    if CP.carFingerprint == CAR.BODY_KNEE:
+      signals += [
+        ("SPEED_L", "KNEE_MOTORS_DATA"),
+        ("SPEED_R", "KNEE_MOTORS_DATA"),
+        ("COUNTER", "KNEE_MOTORS_DATA"),
+        ("CHECKSUM", "KNEE_MOTORS_DATA"),
```

Comment:
what do you use counter and checksum for? they don't need to be added here to validate them in the can parser